skfp/fingerprints/layered.py

```diff
@@ -0,0 +1,50 @@
+from typing import Optional, Sequence, Union
+
+import numpy as np
+from rdkit.Chem import Mol
+from scipy.sparse import csr_array
+
+from skfp.fingerprints.base import FingerprintTransformer
+from skfp.validators import ensure_mols
+
+
+class LayeredFingerprint(FingerprintTransformer):
+    """Pattern fingerprint."""
+
+    def __init__(
+        self,
+        fp_size: int = 2048,
+        min_path: int = 1,
+        max_path: int = 7,
+        branched_paths: bool = True,
+        sparse: bool = False,
+        n_jobs: Optional[int] = None,
+        verbose: int = 0,
+    ):
+        super().__init__(
+            sparse=sparse,
+            n_jobs=n_jobs,
+            verbose=verbose,
+        )
```

Comment:
It might be wise to already add the n_features parameter here from the [set_output pull request](https://github.com/scikit-fingerprints/scikit-fingerprints/pull/113) to avoid changes later

Reply:
This parameter is not available yet. Let's merge that PR first and then this one